package-lock.json

```diff
@@ -20388,67 +20388,6 @@
       "integrity": "sha1-wj6RtxAkKscMN/HhzaknTMOb8vQ=",
       "dev": true
     },
-    "log-symbols": {
```

Comment:
I am guessing this is... OK?

Reply:
It seems like it has cleared up this `UNMET_DEPENDENCY`:
![image](https://user-images.githubusercontent.com/9645462/93212230-3a923880-f75a-11ea-891a-da042dc03b7c.png)
Not entirely sure why it was there in the first place though? It's a dev dependency so I'm not too concerned.

Reply:
It seems like it has cleared up this `UNMET_DEPENDENCY`:
<img width="420" alt="image" src="https://user-images.githubusercontent.com/9645462/93231116-4f79c680-f770-11ea-9ad0-b57128151211.png">
<img width="410" alt="image" src="https://user-images.githubusercontent.com/9645462/93231145-56083e00-f770-11ea-9236-ca3cffa20b5b.png">
Not entirely sure why it was there in the first place though? It's a dev dependency so I'm not too concerned.